src/Sylius/Component/Core/Provider/ProductVariantOriginalPriceMapProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Provider;
+
+use Sylius\Component\Core\Calculator\ProductVariantPricesCalculatorInterface;
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+
+final class ProductVariantOriginalPriceMapProvider implements ProductVariantDataMapProviderInterface
+{
+    public function __construct(private ProductVariantPricesCalculatorInterface $calculator)
+    {
+    }
+
+    public function provide(ProductVariantInterface $variant, ChannelInterface $channel): array
+    {
+        return [
+            'original-price' => $this->calculator->calculateOriginal($variant, ['channel' => $channel]),
```

Comment:
Could we change it to `original_price` without a BC break? For example, passing a parameter as a 2nd argument to the constructor which controls whether the options map key is `original_price` or `original-price`. If the `original-price` is set, depreciation notice will be triggered. WDYT?

Reply:
This is a valid point, but I'd rather it be the other way 'round, since those keys are predominantly (and by default only) used in `data-smth` attributes inside HTML, I'd be for keeping it dashed instead of snake cased, so `applied_promotions` would change to `applied-promotions`.
Adding to that I'd be also for changing the `value` key to `price` since it's more descriptive, but all those changes would necessitate some bc layer inside the template/js possibly duplicating the keys.